instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import CommentSection from '../CommentSection/CommentSection';
+
+
+class PostContainer extends React.Component {
```

Comment:
If you aren't using the state, i'd recommend having a functional component vs class component